### ISSUE TITLE
Technical Story:  Add a11y linting

### ISSUE THREAD

**Issue by MasterEvarior** [puzzle/pcts#33]
## Description
The goal ist to comply with [the WCAG standard](https://www.w3.org/WAI/standards-guidelines/wcag/) as much as feasable. To make this easier, we want to test things (that can be tested in such a way) with a linter. This can include this like:
- alternative description for images
- aria labels
- ...

As usual, this should also be done in a GH Actions workflow, to ensure compliance from every dev.

## Acceptance Criteria
- [x] Linters get automatically executed in the pre-commit hook
- [x] A workflow, which fails when the linter fails, get executed on every push

## Dependencies
- #8 

## Additional Information
- https://angular.dev/best-practices/a11y
- https://dev.to/angular/angular-eslint-rules-for-accessible-html-content-kf5
- https://github.com/orgs/puzzle/projects/17/views/6?pane=issue&itemId=114096721&issue=puzzle%7Cpcts%7C33
- [Some notes to available tools etc](https://codimd.puzzle.ch/JIohrX60QReVbaJGuJYFrw#)
